lib/internal/crypto/random.js

```diff
@@ -16,7 +16,7 @@ function assertOffset(offset, length) {
   }
 
   if (offset > kMaxUint32 || offset < 0) {
-    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'offset', 'uint32');
+    throw new errors.TypeError('ERR_INVALID_ARG_TYPE', 'offset', 'Uint32');
```

Comment:
Tempted to say that uint32 being lowercase was correct... here and in all other instances that aren't referring to Uint32Array.